addons/rearm/functions/fnc_getAllTurrets.sqf

```diff
@@ -0,0 +1,32 @@
+/*
+ * Author: Tuupertunut
+ * Returns all turrets in a vehicle.
+ *
+ * BIS command "allTurrets" does not return the driver turret at the time of writing (2017-07-16).
+ * This function just adds driver turret to the array returned by "allTurrets".
+ *
+ * Arguments:
+ * 0: Vehicle <OBJECT>
+ *
+ * Return Value:
+ * Turret paths <ARRAY>
+ *
+ * Example:
+ * [vehicle] call ace_rearm_fnc_getAllTurrets
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+params [
+    ["_vehicle", objNull, [objNull]]
+];
+
+if (isNull _vehicle) exitWith {[]};
```

Comment:
It probably can not. I only ever added it because some other functions (like [fnc_getMaxMagazines](https://github.com/acemod/ACE3/blob/master/addons/rearm/functions/fnc_getMaxMagazines.sqf)) had it too. I'd be happy to remove it from all of those.